.trivyignore

```diff
@@ -4,3 +4,6 @@ CVE-2024-21538
 
 ## CVE-2024-52798
 CVE-2024-52798
+
+## CVE-2025-22088
+CVE-2025-22088
```

Comment:
I would've expected a new build to have updated the affected package

Reply:
The other images we've had this CVE flag in, we've just essentially built it again, and the `apt-get update` and `apt-get install` have been enough to update the affected package

an `apt-get upgrade` could _maybe_ resolve this